File: www/cgi-bin/get/cookie/file.py
#!/usr/bin/env python3
import cgi
import http.cookies
import os

# Parse incoming cookies
cookies = http.cookies.SimpleCookie(os.environ.get('HTTP_COOKIE', ''))

# Print the Content-Type header
print("Content-Type: text/html\r")
print("\r")

# Print HTML content
print("<html>")
print("<head>")
print("<title>Cookie Test</title>")
print("</head>")
print("<body>")

# Print greeting message
print("<h1>Hello from Python</h1>")
print("<p>Here are the cookies sent with the request:</p>")
print("<ul>")

# Print the names and values of all available cookies
for cookie in cookies.values():
    name = cookie.key
    value = cookie.value
    print(f"<li>{name}: {value}</li>")

print("</ul>")
print("</body>")
print("</html>")

File: www/cgi-bin/hrima/get/cookie/file.php
<?php
// Parse incoming cookies
$cookies = isset($_SERVER['HTTP_COOKIE']) ? $_SERVER['HTTP_COOKIE'] : '';
parse_str($cookies, $parsed_cookies);

// Print the Content-Type header
header('Content-Type: text/html');
echo "\n";

// Print HTML content
echo "<html>\n";
echo "<head>\n";
echo "<title>Cookie Test</title>\n";
echo "</head>\n";
echo "<body>\n";

// Print greeting message
echo "<h1>Hello from PHP</h1>\n";
echo "<p>Here are the cookies sent with the request:</p>\n";
echo "<ul>\n";

// Print the names and values of all available cookies
foreach ($parsed_cookies as $name => $value) {
    echo "<li>$name: $value</li><br>";
}

echo "</ul>\n";
echo "</body>\n";
echo "</html>\n";
?>
